local-env/setup-kind.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# vim:sts=2:ts=2:et:sw=2:tw=0
+
+readonly script_dir=$(dirname "$0")
+readonly kind_cmd="${1}"
+readonly kind_cluster="${2}"
+readonly kind_node_version="${3}"
+readonly kind_kubeconfig="${4}"
+
+export KUBECONFIG="${kind_kubeconfig}"
+
+keycloak_url=https://id.dev.appuio.cloud
+
+step() {
+  echo
+  echo "$1"
+  read -n 1 -s -r -p "Press any key to continue"
+  echo
+}
+
+check_command() {
+  if ! command -v "${1}" >/dev/null 2>&1; then
+    step "Install ${2}. Follow the instructions at ${3}"
+  fi
+}
+
+check_command "kubectl" "kubectl" "https://kubernetes.io/docs/tasks/tools/#kubectl"
+check_command "kubectl-oidc_login" "kubectl oidc-login plugin" "Follow the instructions at https://github.com/int128/kubelogin#setup"
+
+echo
+read -r -p "Provide the URL of the Keycloak to connect the local environment to (default=${keycloak_url}): " user_url
+if [ x"${user_url}" != x"" ]; then
+  keycloak_url="${user_url}"
+fi
+
+echo
+identifier=
+while [ x"$identifier" == x"" ]; do
+  read -r -p "Provide an identifier for your local-dev Keycloak realm: " identifier
+done
+
+realm_name="local-dev-${identifier}"
+sed -e "s/REPLACEME/${realm_name}/g" "${script_dir}/templates/realm.json.tpl" > "${script_dir}/realm.json"
+
+step "Navigate to ${keycloak_url} and create a new realm by importing the '$(realpath "${script_dir}/realm.json")' file."
+
+step "Create a user in the new realm, grant it realm role 'admin', and ensure 'Email Verified' is set to 'On'."
```

Comment:
A note that an email has to be filled in would be great here. I created a user "admin" with no email, but with 'Email Verified' set to On. That is not enough.